test/sdf/unrecognized_elements_with_namespace.sdf

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
```

Comment:
Can we change the version to 1.8 and use an `xmlns` attribute to make the xml valid?
```suggestion
<sdf version="1.8" xmlns:third_party_software="custom_ns_uri">
```

Reply:
Done